corehq/apps/app_manager/util.py

```diff
@@ -545,3 +545,18 @@ def change_unique_id(form):
                 jsonpath_update(reference, id_changes[reference.value])
 
     return app_source
+
+
+def append_update(dict_, new_dict):
```

Comment:
Is this really a more generally useful function or is it just here for the tests?